src/services/session-manager.ts

```diff
@@ -1,4 +1,4 @@
-import { DopplerLegacyClient, DopplerLegacyUserData } from './doppler-legacy-client';
```

Comment:
Is this done automatically? or did you do it yourself?

Reply:
Forget this, I've answered myself reading the commits in detail.

Reply:
I applied that fixes manually because they break our Travis build.

![image](https://user-images.githubusercontent.com/1157864/59704380-5b27b780-91d2-11e9-9cc4-045f726ea2e3.png)

In the future, we need to do something to apply the same validations in our Jenkins build.